src/niclips/io.py

```diff
@@ -13,6 +14,27 @@
 from niclips.image._convert import topil
 from niclips.typing import StrPath
 
+try:
+    import nifti
+
+    HAVE_NIFTI = True
+except ImportError:  # pragma: no cover
+    HAVE_NIFTI = False
+
+
+def load_nifti(fpath: str | Path) -> nib.Nifti1Image:
+    """Wrapper to load Nifti images using library."""
+    if HAVE_NIFTI:
+        hdr, arr = nifti.read_volume(str(fpath))
+        new_hdr = nib.Nifti1Header()
+        for key, val in hdr.items():
+            if key in new_hdr:
+                new_hdr[key] = val
+        aff = new_hdr.get_best_affine()
+        return nib.Nifti1Image(dataobj=arr, affine=aff, header=new_hdr)
```

Comment:
A nibabel compatible interface would be nice to have in nifti proper.

@nx10

Reply:
Also, does `nifti` have tests that show that `nifti` and `nibabel` produce identical data and headers, for a sweep of images (compressed, uncompressed, int type, float type, etc)?

Reply:
read_header should be identical, I have one C function that returns the same header as nibabel (bunch of numpy arrays for even strings and scalar values) and one that returns proper python types (str/int/float/list). I think I use the not-nibabel one in read_volume. No unit tests.